### ISSUE TITLE
Key not found (testing branch ms-support-lv-0.17)

### ISSUE THREAD

**Issue by haslinger** [surface-ui/surface#544]
I am testing the new branch ms-support-lv-0.17 and on the first load(s) of most of the views I get error messages like:

```
[error] #PID<0.770.0> running PanWeb.Endpoint (connection #PID<0.769.0>, stream id 1) terminated
Server: localhost:4000 (http)
Request: GET /
** (exit) an exception was raised:
    ** (KeyError) key :capture_click not found in: %{__changed__: nil, __context__: %{}, __surface__: %{provided_templates: [], slots: %{default: %{size: 0}}}, class: ["hover:text-gray-lighter"], label: "Podcasts new in Panoptikum", to: "/podcasts"}
        (surface 0.7.0-dev) lib/surface/components/utils.ex:89: Surface.Components.Utils.events_to_opts/1
        (surface 0.7.0-dev) lib/surface/components/link.ex:68: Surface.Components.Link.render/1
        (phoenix_live_view 0.17.5) lib/phoenix_live_view/helpers.ex:614: Phoenix.LiveView.Helpers.component/2
        (pan 0.1.0) lib/pan_web/surface/panel.ex:38: anonymous fn/2 in PanWeb.Surface.Panel.render/1
        (phoenix_live_view 0.17.5) lib/phoenix_live_view/diff.ex:387: Phoenix.LiveView.Diff.traverse/7
        (phoenix_live_view 0.17.5) lib/phoenix_live_view/diff.ex:486: anonymous fn/4 in Phoenix.LiveView.Diff.traverse_dynamic/7
        (elixir 1.13.0) lib/enum.ex:2396: Enum."-reduce/3-lists^foldl/2-0-"/3
        (phoenix_live_view 0.17.5) lib/phoenix_live_view/diff.ex:387: Phoenix.LiveView.Diff.traverse/7
        (phoenix_live_view 0.17.5) lib/phoenix_live_view/diff.ex:486: anonymous fn/4 in Phoenix.LiveView.Diff.traverse_dynamic/7
        (elixir 1.13.0) lib/enum.ex:2396: Enum."-reduce/3-lists^foldl/2-0-"/3
        (phoenix_live_view 0.17.5) lib/phoenix_live_view/diff.ex:387: Phoenix.LiveView.Diff.traverse/7
        (phoenix_live_view 0.17.5) lib/phoenix_live_view/diff.ex:138: Phoenix.LiveView.Diff.render/3
        (phoenix_live_view 0.17.5) lib/phoenix_live_view/static.ex:244: Phoenix.LiveView.Static.to_rendered_content_tag/4
        (phoenix_live_view 0.17.5) lib/phoenix_live_view/static.ex:126: Phoenix.LiveView.Static.render/3
        (phoenix_live_view 0.17.5) lib/phoenix_live_view/controller.ex:39: Phoenix.LiveView.Controller.live_render/3
        (phoenix 1.6.5) lib/phoenix/router.ex:355: Phoenix.Router.__call__/2
        (pan 0.1.0) lib/pan_web/endpoint.ex:1: PanWeb.Endpoint.plug_builder_call/2
        (pan 0.1.0) lib/plug/debugger.ex:136: PanWeb.Endpoint."call (overridable 3)"/2
        (pan 0.1.0) lib/pan_web/endpoint.ex:1: PanWeb.Endpoint.call/2
        (phoenix 1.6.5) lib/phoenix/endpoint/cowboy2_handler.ex:54: Phoenix.Endpoint.Cowboy2Handler.init/4
```
with different keys not being found. Interestingly enough, [F5]-reload resolves this, taking several attempts, 'fixing' one missing key per reload. The more LiveComponents involved, the more reloads necessary.
I haven't seen this with latest surface from hex, so I guess it's a regression.

Is there a migration guide, that I could follow for 0.6 -> 0.7 already somewhere? (https://github.com/surface-ui/surface/blob/ms-support-lv-0.17/MIGRATING.md doesn't show anything).

The code of my app [Panoptikum](https://panoptikum.io) is fully open source (although a full setup is very painfull), it is over here in [my gitea](https://code.informatom.com/Panoptikum/pan/src/branch/main). I'm testing in my branch main, that I linked to.

Sorry, that I haven't provided a small app for demoing this yet, I am right within the migration of my big to Tailwind, Alpine, EsBuild, latest Phoenix, and - of course - Surface. 

Happy to give more information, or to try out stuff differently following your suggestions.

## The behavior you expected
The first load should be successfull

## My Environment
Surface: via git, "https://github.com/surface-ui/surface.git", commit: "5a486a4e9dcb96df4027ee76f2387fa6a3c45641", branch: "ms-support-lv-0.17"
LiveView: via git: "https://github.com/phoenixframework/phoenix_live_view.git", commit: "e415eb845cfe2008e1e1ee1494872b296fda567a"
Erlang/OTP 24 [erts-12.1.5] [source] [64-bit] [smp:4:4] [ds:4:4:10] [async-threads:1] [jit]
Elixir 1.13.0 (compiled with Erlang/OTP 24)

**Comment by miguel-s** [comment]
Thanks for reporting @haslinger!

I can confirm that I get the same errors and behavior in one of my apps.

**Comment by msaraiva** [comment]
@haslinger @miguel-s this has been fixed in 97194adad418c54cf26a4fe12080f9098547265b.

Please let me know if you still run into any issues after pulling those changes.

Thanks for reporting this issue! ❤️

**Comment by haslinger** [comment]
No issues for me any more! Thank you!
